server/src/main/antlr4/com/broadcom/lsp/cobol/core/parser/CobolLexer.g4

```diff
@@ -15,9 +15,59 @@
 lexer grammar CobolLexer;
 import CICSLexer;
 channels{TECHNICAL}
+@header {
+  import java.util.regex.Matcher;
+  import java.util.regex.Pattern;
+  import org.apache.commons.lang3.StringUtils;
+}
+@lexer::members {
+private Pattern titlePattern = Pattern.compile("TITLE\\s*(.*?)\\.?\\s\\n.*", Pattern.CASE_INSENSITIVE);
```

Comment:
Making 'titlePattern' static final, will be more performant (pattern compiled only once).